ekf:
  ros__parameters:
    NODE_SEQUENCE: [10, 4, 20, 10]

    ENCODER_COUNTS_PER_REV: 720.0 # counts per revolution
    WHEEL_RADIUS: 0.033 # front/rear wheel radius in m
    PIN_TO_SPUR_RATIO: 0.09536679536679536
    VIRTUAL: false

vehicle_control_ros:
  ros__parameters:
    v_ref: 0.5
    K_p: 0.1
    K_i: 1
    K_stanley: 1
    NODE_SEQUENCE: [10, 4, 10]

    ENCODER_COUNTS_PER_REV: 720.0 # counts per revolution
    WHEEL_RADIUS: 0.033 # front/rear wheel radius in m
    PIN_TO_SPUR_RATIO: 0.09536679536679536
    VIRTUAL: false

vehicle_control_ros_bridge1:
  ros__parameters:
    v_ref: 0.9
    K_p: 0.1
    K_i: 1
    K_stanley: 1
    NODE_SEQUENCE: [10, 4, 20, 10]

    ENCODER_COUNTS_PER_REV: 720.0 # counts per revolution
    WHEEL_RADIUS: 0.033 # front/rear wheel radius in m
    PIN_TO_SPUR_RATIO: 0.09536679536679536
    qcar_id: 1

vehicle_control_ros_bridge2:
  ros__parameters:
    v_ref: 0.9
    K_p: 0.1
    K_i: 1
    K_stanley: 1
    NODE_SEQUENCE: [10, 4, 20, 10]

    ENCODER_COUNTS_PER_REV: 720.0 # counts per revolution
    WHEEL_RADIUS: 0.033 # front/rear wheel radius in m
    PIN_TO_SPUR_RATIO: 0.09536679536679536
    qcar_id: 2

vehicle_control_ros_bridge3:
  ros__parameters:
    v_ref: 0.9
    K_p: 0.1
    K_i: 1
    K_stanley: 1
    NODE_SEQUENCE: [10, 4, 20, 10]

    ENCODER_COUNTS_PER_REV: 720.0 # counts per revolution
    WHEEL_RADIUS: 0.033 # front/rear wheel radius in m
    PIN_TO_SPUR_RATIO: 0.09536679536679536
    qcar_id: 3

lidar_occupancy_node:
  ros__parameters:
    CELLS_PER_METER: 20
    GRID_WIDTH_METERS: 6
    GRID_HEIGHT_METERS: 3
    IS_OCCUPIED: 100
    IS_FREE: 50
    VIRTUAL: false